design/FY2022/NFP-RefCondsWtrToAirCoils.md

```diff
@@ -0,0 +1,283 @@
+
+### Example ###
+
+The following example illustrates the issue and proposed approach. Let's consider a model using a `Coil:Cooling:WaterToAirHeatPump:EquationFit` using the same performance curves as in the `ASHRAE901_ApartmentHighRise_STD2019_Denver.idf` file.
+
+Let's assume that:
+- The coil load at peak design conditions is `CoolCapAtPeak` = 1000W, 
+- The entering air wet-bulb temperature at peak design conditions is `MixWetBulb` = 15.6&deg;C, 
+- The entering air dry-bulb temperature at peak design conditions is `MixTemp` = 21.1&deg;C, 
+- The user-specified cooling COP at rated conditions is `RatedCOPCool` = 4.2
+
+#### Current Approach ###
+_Note: The current variable names are used below._
+
+- `PeakTotCapTempModFac` is calculated at 29.44&deg;C (85&deg;F) EWT and 15.6&deg;C (60&deg;F) EAWT and is about 0.873
+- `RatedCapCoolTotalDes` = 1000 / 0.873 = 1145 W
+- `RatedPowerCool` = 1145 / 4.2 = 273 W
+
+The rating conditions for water-to-air water loop heat pumps are 30&deg;C (86&deg;F) EWT and 19&deg;C (66.2&deg;F), at these conditions the `TotCapTempModFac` is 0.956 and `PowerTempModFac` is 1.006. When using these modifiers to calculate the operating capacity and power at these conditions we get a capacity of 1145 * 0.956 ~= 1095 W and a power of 274 * 1.006 ~= 274 W which correspond to a COP of 1095 / 274 ~= 4 which doesn't align with the user-specified COP of 4.2.
+
+#### Proposed Approach ####
+_Note: The proposed variable names are used below._
+
+- `PeakTotCapTempModFac` is calculated at 30&deg;C (86&deg;F) EWT and 15.6&deg;C (60&deg;F) EAWT and is about 0.869
+- `RefTotCapTempModFac` is calculated at 30&deg;C (86&deg;F) EWT and 19&deg;C (66.2&deg;F) EAWT and is about 0.956 which corresponds to the total capacity modifier at the reference conditions which are the actual rating condition in this example.
+- `RefPowerTempModFac` is calculated at 30&deg;C (86&deg;F) EWT and 19&deg;C (66.2&deg;F) EAWT and is about 1.006 which corresponds to the power modifier at the reference conditions which are the actual rating condition in this example.
+- `RefCapCoolTotalDes` = 1000 / 0.869 = 1151 W
+- `RatedPowerCool` = 1151 * 0.956 / (4.2 * 1.006) = 260 W
+
+The rating conditions for water-to-air water loop heat pumps are 30&deg;C (86&deg;F) EWT and 19&deg;C (66.2&deg;F), at these conditions the `TotCapTempModFac` is 0.956 and `PowerTempModFac` is 1.006. When using these modifiers to calculate the operating capacity and power at these conditions we get a capacity (`CapCoolTotalDesAtRefCdts`) of 1151 * 0.956 = 1101 W and a power (`PowerCoolAtRefCdts`) of 260 * 1.006 ~= 262 W which correspond to a COP of 1101 / 262 ~= 4.2 which aligns with the user-specified COP of 4.2.
+
```

Comment:
If you know you need a coil capacity of 1000 W, and the perf curve says you need 1145 W at the coil inlet conditions, why don't you just calculate power using 1145 / (4.2*1.006) = 270.993 W. Doesn't that give you a COP of 4.2 at the rating  point? Or are these performance curves not representative of a WSHP?